scripts/pre_commit_linter.py

```diff
@@ -140,7 +149,9 @@ def _pre_commit_linter():
     and pass JSCS binary path
     """
     parsed_args = _PARSER.parse_args()
-    autofix = bool(parsed_args.autofix)
+    input_dir = None
+    if len(parsed_args.directory):
+        input_dir = os.path.join(os.getcwd(), parsed_args.directory[0])
```

Comment:
I don't mind either way, but if we were just allowing linting a single folder, why treat the argument as a list instead of a string?